rnef.config.mjs

```diff
@@ -3,13 +3,17 @@ import {platformIOS} from '@rnef/platform-ios';
 import {pluginMetro} from '@rnef/plugin-metro';
 
 const isHybrid = process.env.IS_HYBRID_APP === 'true';
+const hybridRoot = './Mobile-Expensify/';
 
 /** @type {import('@rnef/config').Config} */
 export default {
     remoteCacheProvider: null,
     bundler: pluginMetro(),
     platforms: {
-        ios: platformIOS({sourceDir: isHybrid ? './Mobile-Expensify/iOS' : './ios'}),
-        android: platformAndroid({sourceDir: isHybrid ? './Mobile-Expensify/Android' : './android'}),
+        ios: platformIOS({sourceDir: isHybrid ? `${hybridRoot}/iOS` : './ios'}),
+        android: platformAndroid({sourceDir: isHybrid ? `${hybridRoot}/Android` : './android'}),
+    },
+    fingerprint: {
+        extraSources: isHybrid ? [`${hybridRoot}/iOS`, `${hybridRoot}/Android`, `${hybridRoot}/patches`] : [],
```

Comment:
Now this is not ideal, because we'll eg. rebuild iOS (Hybrid) if Native Android (Hybrid) changes. It's a unique scenario and should not bite us in any way, might only slightly lower the cache hit ratio in these specific scenarios. I'll monitor of this is the case and we should contribute back even more granularity.

Reply:
I think we can live with lower cache hit ratios for now, but in the future it seems like this could be improved upstream with the introduction of platform-specific fingerprint configs like `extraSourcesIOS` and `extraSourcesAndroid`

Reply:
I'm not sure how @parasharrajat's suggestion would improve upon the limitation of this code, unless we created separate hashes for iOS and Android. But that's essentially a fingerprinting layer on top of a fingerprinting layer, which just feels complex. Understanding that HybridApp presents some unique challenges, I'd love to minimalize the customization we have on top of rnef. The configs throughout this repo are already quite complex.